openlibrary/templates/type/author/view.html

```diff
@@ -91,7 +91,7 @@ <h2 class="author collapse">
 
     $if ctx.user and ("merge-authors" in ctx.features or ctx.user.is_admin()) and query_param('merge', 'false') == 'true':
        <div class="message hidden">
```

Comment:
@jdlrobson I have tested and am not seeing the progress or messages, I think this `hidden` is overriding the visibility of the sub elements? If I remove the `hidden` manually after the page has loaded, I do see just the final status message I would expect to see.

Reply:
Looks like i missed one. This should also be style="display:none"

Is there any easy way for me to get this interface locally. I'd love to be more helpful and test this.